src/analysis/inner_product_space/projection.lean

```diff
@@ -772,6 +772,23 @@ lemma orthogonal_projection_mem_subspace_orthogonal_complement_eq_zero
   orthogonal_projection K v = 0 :=
 by { ext, convert eq_orthogonal_projection_of_mem_orthogonal _ _; simp [hv] }
 
+lemma orthogonal_projection_eq_linear_proj [complete_space K] :
```

Comment:
I think `K` should be implicit in this lemma, and it should be called something like
```suggestion
lemma orthogonal_projection_coe_linear_map_eq_linear_proj [complete_space K] :
```